exercises/affine-cipher/example.R

```diff
@@ -0,0 +1,63 @@
+# encrypts message
+encrypt <- function(message, a, b) {
+  m <- 26
+
+  # computes the greatest common divisor of numbers x and y
+  gcd <- function(x, y) {
+    r <- x %% y
+    return(ifelse(r, gcd(y, r), y))
+  }
+
+  # must check a and m are coprime
+  if (gcd(a, m) != 1) {
+    stop("a and 26 must be co-prime")
```

Comment:
Should `26` be hard-coded here? I think `stop` can act like `paste` in that `m` could be inserted dynamically here.

Reply:
Yes okay